src/components/Provider/Analytics/ProviderStats/StatsFilter/StatsFilter.js

```diff
@@ -23,11 +24,20 @@ import { useIntl } from "react-intl";
  * @returns {JSX.Element} JSX Component.
  */
 const StatsFilter = ({ list, onChange }) => {
+  const baseExchanges = useExchangeList();
   const [exchange, setExchange] = useState("ALL");
   const [quote, setQuote] = useState("ALL");
   const intl = useIntl();
   const exchanges = useExchangesOptions(true);
-  const quoteAssets = useQuoteAssets();
+  const selectedBase =
```

Comment:
maybe `selectedExchange`?

Reply:
selectedExchange is confusing with the store's selectedExchange, maybe should be selectedBaseExchange, or selectedParentExchange.

Reply:
selectedExchangeFilter then ;)

Reply:
@cwagner22  done.